docker-compose.yml

```diff
@@ -66,6 +66,8 @@ services:
       - DATASTORE_USE_PROJECT_ID_AS_APP_ID=true
       - GOOGLE_CLOUD_PROJECT=dev-project-id
       - REDIS_HOST=redis
+    labels:
+      - COMPLETED_FIRST_RUN=${COMPLETED_FIRST_RUN:-1}
```

Comment:
```suggestion
      - COMPLETED_FIRST_RUN=${COMPLETED_FIRST_RUN:0}
```

Reply:
almost done! need to figure out other things before confirming.